### PR TITLE
Example: month

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: Tasks/month.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Get month number
+
+Months = [
+'jan','feb','mar','apr','may','jun','jul','aug','sep','oct','nov','dec'
```
Comment (DenisMilkevich) [inline Tasks/month.js:4]: On the top we can add 'use string'

Comment (DenisMilkevich) [inline Tasks/month.js:4]: need to be const months (using LoverCase for variables)

File: Tasks/month.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+]
```
Comment (DenisMilkevich) [inline Tasks/month.js:5]: Need to be const months (using LoverCase for variables)

File: Tasks/month.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+Month=s=>{
```
Comment (DenisMilkevich) [inline Tasks/month.js:7]: Ditto

File: Tasks/month.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  l=Months.length
+    for(let i=0;i<l;i++){;
+      if(s.toLowerCase().startsWith(Months[i]))return(i+1)
```
Comment (DenisMilkevich) [inline Tasks/month.js:10]: We can make 'toLoverCase' a new variable for better reading

File: Tasks/month.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+        };
+          return -1
+            }
+
+const result = Month('february')
```
Comment (DenisMilkevich) [inline Tasks/month.js:15]: Month toLowerCase

File: Tasks/month.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+Months = [
+'jan','feb','mar','apr','may','jun','jul','aug','sep','oct','nov','dec'
+]
+
+Month=s=>{
+  l=Months.length
+    for(let i=0;i<l;i++){;
+      if(s.toLowerCase().startsWith(Months[i]))return(i+1)
+        };
+          return -1
```
Comment (DenisMilkevich) [inline Tasks/month.js:12]: we don't need '-' near '1' hear

File: Tasks/month.js (hunk 4 above)
Comment (DenisMilkevich) [inline Tasks/month.js:10]: Month toLowerCase

File: Tasks/month.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+            }
+
+const result = Month('february')
+console.log(result)
```
Comment (DenisMilkevich) [inline Tasks/month.js:16]: I have added "const' also restructure some code and rewrite variables to lover case

File: Tasks/month.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
+// Get month number
+
+Months = [
+'jan','feb','mar','apr','may','jun','jul','aug','sep','oct','nov','dec'
+]
+
+Month=s=>{
+  l=Months.length
```
Comment (DenisMilkevich) [inline Tasks/month.js:8]: Here we need 'const' near 'l'

File: Tasks/month.js [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+    for(let i=0;i<l;i++){;
+      if(s.toLowerCase().startsWith(Months[i]))return(i+1)
+        };
+          return -1
+            }
```
Comment (DenisMilkevich) [inline Tasks/month.js:13]:
```js
const result = Month('february')
```

**Comment by tshemsedinov** [comment]
@DenisMilkevich Comments are clear, but please provide a refactored code and python translation

File: Tasks/month.js (hunk 7 above)
Comment (DenisMilkevich) [inline Tasks/month.js:16]:
```js
'use strict';

const months = ([
    "jan",
    "feb",
    "mar",
    "apr",
    "may",
    "jun",
    "jul",
    "aug",
    "sep",
    "oct",
    "nov",
    "dec",
]);

const month = (s) => {
    const lovered = s.toLowerCase();
    let l = months.length;
    for (let i = 0; i < l; i++) {
        if (lovered.startsWith(months[i]))
            return i + 1;

    }
    return l;

};

const result = month('April');
console.log(result);
```

File: Tasks/month.js (hunk 3 above)
Comment (DenisMilkevich) [inline Tasks/month.js:7]: I think in func name we need to ride some action so 'matchMonths' is better

File: Tasks/month.js [hunk 10]
```diff
@@ -0,0 +1,16 @@
+// Get month number
+
+Months = [
+'jan','feb','mar','apr','may','jun','jul','aug','sep','oct','nov','dec'
+]
+
+Month=s=>{
+  l=Months.length
+    for(let i=0;i<l;i++){;
```
Comment (DenisMilkevich) [inline Tasks/month.js:9]: We don't need for loop here, just can add .indexOf

File: Tasks/month.js (hunk 7 above)
Comment (DenisMilkevich) [inline Tasks/month.js:16]:
```py
months = ([
    "jan",
    "feb",
    "mar",
    "apr",
    "may",
    "jun",
    "jul",
    "aug",
    "sep",
    "oct",
    "nov",
    "dec",
])


def match_month(item):
    lovered = item.lower()
    arr = months.index(lovered[0:3])
    return arr + 1


result = match_month('february')
print(result)
```

**Pull request closed without merging**